Gruntfile.js

```diff
@@ -45,6 +45,8 @@ module.exports = function (grunt) {
         ]
       },
       dist: {
+        // TODO: edit this task to work with 'docroot/**/humsci/**/*.css'
```

Comment:
Actually, can we add a "why" in here? Like "this glob doesn't currently include all our css files".